editor/sidebar/block-inspector/advanced-controls.js

```diff
@@ -64,20 +65,22 @@ class BlockInspectorAdvancedControls extends Component {
 	render() {
 		const { selectedBlock, post } = this.props;
 		const blockType = getBlockType( selectedBlock.name );
-		if ( false === blockType.className && ! blockType.supportAnchor ) {
+		const supportAnchor = blockType.support && !! blockType.support.anchor;
+		const supportCustomClassname = isUndefined( get( blockType, 'support.className' ) ) || blockType.support.className;
```

Comment:
Seems like the previous implementation was simpler? `false !== support.className`